headsup.sql

```diff
@@ -8,6 +8,13 @@ CREATE TABLE notes (
     PRIMARY KEY(id)
 );
 
+CREATE TABLE dispatchMishaps (
```

Comment:
Please rename this to `dispatch_mishaps` to fall into line with conventions in other RPIA projects (namely the new rpia3.0)

Reply:
Completed.